### PR TITLE
Ria 1647 end appeal notify ho

### PR DESCRIPTION
**Pull request by makowalski**
### JIRA link (if applicable) ###
[RIA-1647](https://tools.hmcts.net/jira/browse/RIA-1647)

### Change description ###
end appeal notify ho

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -19,10 +19,13 @@ List<LinkedHashMap<String, Object>> secrets = [
     secret('respondent-evidence-direction-email', 'IA_RESPONDENT_EVIDENCE_DIRECTION_EMAIL'),
     secret('respondent-review-direction-email', 'IA_RESPONDENT_REVIEW_DIRECTION_EMAIL'),
     secret('respondent-non-standard-direction-until-listing-email', 'IA_RESPONDENT_NON_STANDARD_DIRECTION_UNTIL_LISTING_EMAIL'),
+    secret('home-office-end-appeal-email', ' IA_HOME_OFFICE_END_APPEAL_EMAIL'),
```
Comment (bradsorour) [inline Jenkinsfile_CNP]: Whitespace before ' IA_HOME...

File: Jenkinsfile_nightly [hunk 2]
```diff
@@ -23,10 +23,13 @@ List<LinkedHashMap<String, Object>> secrets = [
     secret('respondent-evidence-direction-email', 'IA_RESPONDENT_EVIDENCE_DIRECTION_EMAIL'),
     secret('respondent-review-direction-email', 'IA_RESPONDENT_REVIEW_DIRECTION_EMAIL'),
     secret('respondent-non-standard-direction-until-listing-email', 'IA_RESPONDENT_NON_STANDARD_DIRECTION_UNTIL_LISTING_EMAIL'),
+    secret('home-office-end-appeal-email', ' IA_HOME_OFFICE_END_APPEAL_EMAIL'),
```
Comment (bradsorour) [inline Jenkinsfile_nightly]: Same as above

File: Jenkinsfile_parameterized [hunk 3]
```diff
@@ -22,6 +22,7 @@ List<LinkedHashMap<String, Object>> secrets = [
     secret('respondent-evidence-direction-email', 'IA_RESPONDENT_EVIDENCE_DIRECTION_EMAIL'),
     secret('respondent-review-direction-email', 'IA_RESPONDENT_REVIEW_DIRECTION_EMAIL'),
     secret('respondent-non-standard-direction-until-listing-email', 'IA_RESPONDENT_NON_STANDARD_DIRECTION_UNTIL_LISTING_EMAIL'),
+    secret('home-office-end-appeal-email', ' IA_HOME_OFFICE_END_APPEAL_EMAIL'),
```
Comment (bradsorour) [inline Jenkinsfile_parameterized]: Same as above

File: Jenkinsfile_CNP (hunk 1 above)
Comment (makowalski) [reply Jenkinsfile_CNP]: Thanks

**Pull request merged**